Subject: set proper path for qemu-bridge-helper binary in the docs
Bug-Debian: https://bugs.debian.org/1027447
From: Michael Tokarev <mjt@tls.msk.ru>
Date: Tue, 18 Apr 2023 17:29:17 +0300
Forwarded: no
Upstream-Status: this should be fixed in a more generic way, this fix is debian-specific

Upstream should propagate this path from the config option.
This is a dumb-n-quick fix.

diff --git a/qemu-options.hx b/qemu-options.hx
index b56f6b2fb2..fe138c83d3 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -3158,3 +3158,3 @@ SRST
     The default network helper executable is
-    ``/path/to/qemu-bridge-helper`` and the default bridge device is
+    ``/usr/lib/qemu/qemu-bridge-helper`` and the default bridge device is
     ``br0``.
@@ -3184,3 +3184,3 @@ SRST
         |qemu_system| linux.img -device virtio-net-pci,netdev=n1 \\
-                -netdev tap,id=n1,"helper=/path/to/qemu-bridge-helper"
+                -netdev tap,id=n1,"helper=/usr/lib/qemu/qemu-bridge-helper"
 
@@ -3191,3 +3191,3 @@ SRST
     attach it to the bridge. The default network helper executable is
-    ``/path/to/qemu-bridge-helper`` and the default bridge device is
+    ``/usr/lib/qemu/qemu-bridge-helper`` and the default bridge device is
     ``br0``.
